ndcube/ndcube_sequence.py

```diff
@@ -131,37 +131,19 @@ def cube_like_world_axis_physical_types(self):
 
     @property
     def common_axis_extra_coords(self):
-        common_extra_coords = None
-        if self._common_axis in range(self.data[0].wcs.naxis):
-            if self.data[0].extra_coords:
-                common_extra_coords = {}
-                coord_names = list(self.data[0].extra_coords.keys())
-                for coord_name in coord_names:
-                    if self.data[0].extra_coords[coord_name]["axis"] == self._common_axis:
-                        try:
-                            coord_unit = self.data[0].extra_coords[coord_name]["value"].unit
-                            qs = tuple([np.asarray(
-                                c.extra_coords[coord_name]["value"].to(coord_unit).value)
-                                        for c in self.data])
-                            common_extra_coords[coord_name] = u.Quantity(np.concatenate(qs),
-                                                                         unit=coord_unit)
-                        except AttributeError:
-                            qs = tuple([np.asarray(c.extra_coords[coord_name]["value"])
-                                        for c in self.data])
-                            common_extra_coords[coord_name] = np.concatenate(qs)
-        return common_extra_coords
+        if not isinstance(self._common_axis, int):
```

Comment:
do we always cast to an integer when setting `self._common_axis`? Otherwise this could end up as a float?

Reply:
Good point!  No we don't  I'll add that into the ```NDCubeSequence.__init__```.